### ISSUE TITLE
训练CTR-GCN，训练完一个epoch，更新学习率时报错

### ISSUE THREAD

**Issue by jiaerwang0328** [PaddlePaddle/PaddleVideo#541]
执行python main.py --validate -c configs/recognition/ctrgcn/ctrgcn_ntucs_joint_.yaml --seed 1 时报错：
![image](https://user-images.githubusercontent.com/38750407/194795127-0c7f1ba6-957f-45d7-a81e-7f14024ce600.png)

**Comment by huangjun12** [comment]
这个报错表示 `self.num_iters`为None, `self.num_iters`是[这行](https://github.com/PaddlePaddle/PaddleVideo/blob/714bde8c80ffd223869337f8c8fcc32b16dada3b/paddlevideo/tasks/train.py#L134) 里的`len(train_loader)`传进去的，可以打印下`len(train_loader)`是否为None，定位下它为什么是None，可能是数据集路径不对？

**Comment by jiaerwang0328** [comment]
> 这个报错表示 `self.num_iters`为None, `self.num_iters`是[这行](https://github.com/PaddlePaddle/PaddleVideo/blob/714bde8c80ffd223869337f8c8fcc32b16dada3b/paddlevideo/tasks/train.py#L134) 里的`len(train_loader)`传进去的，可以打印下`len(train_loader)`是否为None，定位下它为什么是None，可能是数据集路径不对？

你好，数据集路径是对的，len(train_loader)不为None。

**Comment by huangjun12** [comment]
https://github.com/PaddlePaddle/PaddleVideo/blob/714bde8c80ffd223869337f8c8fcc32b16dada3b/paddlevideo/solver/custom_lr.py#L297 这一行改为`num_iters=1,`
